integration_test/cases/preload.exs

```diff
@@ -35,8 +35,8 @@ defmodule Ecto.Integration.PreloadTest do
 
     # With assoc query
     assert [p3, p1, p2] = TestRepo.preload([p3, p1, p2], :comments)
-    assert [%Comment{id: ^cid1}, %Comment{id: ^cid2}] = p1.comments
-    assert [%Comment{id: ^cid3}, %Comment{id: ^cid4}] = p2.comments
+    assert [%Comment{id: ^cid1}, %Comment{id: ^cid2}] = p1.comments |> Enum.sort_by(&(&1.id))
```

Comment:
Let's have a `sort_by_id` function in this module in special since we use it so many times...